doc/api/assert.md

```diff
@@ -348,7 +348,7 @@ const callsfunc = tracker.calls(func);
 callsfunc(1, 2, 3);
 
 assert.deepStrictEqual(tracker.getCalls(callsfunc),
-                       [{ thisArg: this, arguments: [1, 2, 3 ] }]);
+                       [{ thisArg: this, arguments: [1, 2, 3] }]);
```

Comment:
Let's use `undefined` here, as I said in my previous comment, while `this` happens to be technically correct here (because we're at the top-level of an ES module), folks who are going to copy this snippet may not be and could be surprised by the result, or it could add to the general confusion of how `this` bindings work in JS. Let's be explicit.
```suggestion
                       [{ thisArg: undefined, arguments: [1, 2, 3] }]);
```

Reply:
I misunderstand previous comment. Applied your suggestion again.